{
  "0.11": {
    "type": "catalogRender",
    "schema": 1,
    "eval": {
      "version": "0.11",
      "pname": "pyditz",
      "name": "pyditz-0.11",
      "system": "aarch64-linux",
      "outputs": {
        "out": "/nix/store/bmggjyfhbcsqifzj1h2dzzpd73gwwpyc-pyditz-0.11"
      },
      "meta": {
        "outputsToInstall": [
          "out"
        ],
        "description": "Drop-in replacement for the Ditz distributed issue tracker",
        "position": "/nix/store/5g0bnn21bqpd241da9yyivjkcfy0ra1w-source/pkgs/applications/misc/pyditz/default.nix:24",
        "unfree": false
      }
    },
    "element": {
      "attrPath": "legacyPackages.aarch64-linux.staging.pyditz",
      "originalUrl": "flake:floxpkgs",
      "outputs": null,
      "storePaths": [
        "/nix/store/bmggjyfhbcsqifzj1h2dzzpd73gwwpyc-pyditz-0.11"
      ],
      "url": "github:flox/floxpkgs/e0169d7a9d324afebf5679551407756c77af8930"
    },
    "source": {
      "locked": {
        "revCount": 384129,
        "lastModified": 1654682581
      }
    }
  }
}
